provider/marathon/marathon.go

```diff
@@ -420,6 +444,193 @@ func (p *Provider) hasCircuitBreakerLabels(application marathon.Application) boo
 	return ok
 }
 
+// Map of services properties
+// we can get it with label[serviceName][propertyName] and we got the propertyValue
+type serviceProperties map[string]string
+
+type labelServiceProperties map[string]serviceProperties
+
+// Check if for the given task and applications, we find labels that are defining services
+func (p *Provider) hasTaskServices(task marathon.Task, applications []marathon.Application) bool {
+	application, err := getApplication(task, applications)
+	if err != nil {
+		log.Errorf("Unable to get marathon application from task %s", task.AppID)
+		return false
+	}
+	return p.hasServices(application)
+}
+
+// Check if for the given application, we find labels that are defining services
+func (p *Provider) hasServices(application marathon.Application) bool {
+	return len(extractServicesLabels(application.Labels)) > 0
+}
+
+// Extract the service labels from application labels of Application struct
+func extractServicesLabels(labels *map[string]string) labelServiceProperties {
+	v := make(labelServiceProperties)
+
+	if labels != nil {
+		for label, value := range *labels {
+			matches := servicesPropertiesRegexp.FindStringSubmatch(label)
+			if matches == nil {
+				continue
+			}
+
+			result := make(map[string]string)
```

Comment:
Here's the promised comment I announced in another comment:

I don't think we need the `result` map. Our regular expression pattern basically guarantees us that we'll have `service_name` and `property_name` sub-matches. So we can just extract the names/values at the known index positions and use them to populate the `labelServiceProperties` map.